style: apply /style-guide pass to weave (#2739)
wandb/docs: weave/cookbooks/custom_model_cost.mdx

weave_client = weave.init(name_of_wandb_project)
```

### Set up a model with Weave

> Next, define a Weave `Model` subclass that performs its own token accounting. Returning the usage counts and model name in the output dictionary is what lets Weave later associate the call with the custom cost you'll define in the next section.

Reviewer: ```suggestion
Next, define a Weave `Model` subclass that performs its own token accounting. Returning the usage counts and model name in the output dictionary is what lets Weave later associate the call with the custom cost defined in the next section.
```

Author: Applied verbatim in 558ba397a.

        return outputs["output"]
```

### Add a custom cost

> Add a custom cost. Now that you have a custom cost and your calls have usage, you can fetch the calls with `include_cost` and your calls will have costs under `summary.weave.costs`.

Reviewer: ```suggestion
Add a custom cost. Once you have added a custom cost and your calls have usage, you can fetch the calls with `include_cost` and review the costs under `summary.weave.costs`.
```

Author: Applied verbatim in 558ba397a.
